lib/base/serialize.js

```diff
@@ -98,7 +101,7 @@ module.exports.prefixedCollapsibleMap = function(m, prefix) {
  * @returns {string|object} stringified object
  */
 module.exports.object = function(o) {
-  if (_.isObject(o) || _.isArray(o)) {
+  if (typeof o === 'object' || typeof o === 'array') {
```

Comment:
An array is an object so if o is was an array, the `typeof o` would return 'object'.  `if (typeof o === 'object')` should be sufficient